### PR TITLE
release raiwidgets 0.2.1

### PR DESCRIPTION
**Pull request by imatiach-msft**
updated release notes for 0.2.1 release of raiwidgets package to pypi

## v0.2.1

Update to `raiwidgets` package.

- educational materials:
  - add imports for `MimicExplainer` and `PFIExplainer`
- features

  - migrate cohort and dataset utilities to core-ui
  - remove circular dependencies
  - convert flask comms class to function
  - add confirmation popup in `ExplanationDashboard` when cancelling editing
    a cohort.
  - fix clipped dropdown in `FairnessDashboard`
  - keep `FairnessDashboard` chart state in single model view
  - in large data view, change `ErrorAnalysisDashboard` cohort state to use
    full data instead of downsampled explanation data
  - updates to `ErrorAnalysisDashboard`:
    - add true y and predicted y values
    - fix categorical labels handling in what if panel
    - add breadcrumb to error explorer view and minor style adjustments to
      navigation panel
    - fix indexing issues for all data cohort
    - add color column to local explanation view

- pypi:

  - raiwidgets: release v0.2.1

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -16,14 +16,43 @@ Note that it is not required to have an entry for every pull request.
...
+    - add true y and predicted y values
+    - fix categorical labels handling in what if panel
+    - add breadcrumb to error explorer view and minor style adjustments to
+      navigation panel
+    - fix indexing issues for all data cohort
+    - add color column to local explanation view
+
+- pypi:
+
+  - raiwidgets: release v0.2.1
```
Comment (romanlutz) [inline CHANGES.md:54]: What's this for? It already says 0.2.1 at the top.

Also, if this was a minor release (e.g., 0.3.0) then we'd need to increase the version in the npm packages. But it isn't, so just FYI

File: CHANGES.md [hunk 2]
```diff
@@ -16,14 +16,43 @@ Note that it is not required to have an entry for every pull request.
...
 - breaking changes
 - bug fixes
 - other
 
+## v0.2.1
+
+Update to `raiwidgets` package.
+
+- educational materials:
+  - add imports for `MimicExplainer` and `PFIExplainer`
```
Comment (romanlutz) [inline CHANGES.md:32]: Is this in a notebook? Would be nice to have it clarified.

File: CHANGES.md [hunk 3]
```diff
@@ -16,14 +16,43 @@ Note that it is not required to have an entry for every pull request.
...
+- features
+
+  - migrate cohort and dataset utilities to core-ui
+  - remove circular dependencies
+  - convert flask comms class to function
```
Comment (romanlutz) [inline CHANGES.md:37]: This probably isn't worth mentioning, no real functionality change.

**Review by romanlutz — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (imatiach-msft) [reply CHANGES.md:54]: done, removed!

File: CHANGES.md (hunk 2 above)
Comment (imatiach-msft) [reply CHANGES.md:32]: done!

File: CHANGES.md (hunk 3 above)
Comment (imatiach-msft) [reply CHANGES.md:37]: removed "convert flask comms class to function"

**Pull request merged**